bin/fin

```diff
@@ -4264,6 +4276,29 @@ update ()
 	fi
 }
 
+# Return latest tag's commit url
+get_latest_release ()
+{
+	# Get the latest release tag
+	URL_REPO_RELEASE="${GITHUB_API}/repos/${REPO_URL}/releases/latest"
+	local latest_tag=$(curl -sSL $URL_REPO_RELEASE | grep tag_name | sed 's/\"tag_name": \"\(.*\)",/\1/' | tr -d '[:space:]')
+
+	# Get information about that tag as we need to get the sha tag associated with that
+	URL_REPO_TAG="${GITHUB_API}/repos/${REPO_URL}/git/refs/tags/${latest_tag}"
+	local commit_sha=$(curl -sSL $URL_REPO_TAG | grep sha | sed 's/\"sha": \"\(.*\)",/\1/' | tr -d '[:space:]')
```

Comment:
Why is there a need to get commit? You can just use tag name

```bash
https://raw.githubusercontent.com/docksal/docksal/${latest_tag}/bin/fin
```

Reply:
@achekulaev documentation suggested I needed to get the SHA in order to get the file